yt/python/yt/environment/api/__init__.py

```diff
@@ -48,6 +48,7 @@ class LocalYtConfig(object):
         "sharding_key_length": 1,
     })
     default_docker_image = "docker.io/library/python:2.7-slim"
+    job_proxy_log_manager_directory_traversal_concurrency = attr.ib(None)
```

Comment:
Why do we need such a concrete option here?

It looks like for configuration can be just used `job_proxy_log_manager` config